tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -23,6 +22,55 @@ public NodeAngularAppTest(ITestOutputHelper output, TestTempDirTestFixture fixtu
 
         public const int PortInContainer = 4200;
 
+        /// <summary>
+        /// IMPORTANT:
+        /// New tests should be included in a corresponding
+        /// method with attribute:
+        ///     [Fact, Trait("category", "<platform>-<version>")]
+        ///
+        /// The pipeline will invoke these integration tests 
+        /// on the matching category attribute.
+        /// </summary>
+        [Fact, Trait("category", "node-8")]
+        public void PipelineTestInvocationsNode8()
+        {
+            string nodeVersion = "8";
+            CanBuildAndRun_Angular6App_WithoutCompressedNodeModules(nodeVersion);
```

Comment:
At this point, do we even need each function (`CanBuildAndRun_Angular6App_WithoutCompressedNodeModules`, etc.) to be a test? Should they just be a simple function without Theory or InlineData tags?

Reply:
+1 to this -- it seems we're running these functions twice, so unless there's a scenario we're looking to cover where we run the functions one after another, then we should be able to remove the `Theory`/`InlineData` attribute from the functions themselves so that only the pipeline tests functions are called.

Reply:
That's true. I'll look either:

- removing theory / inlineData for every test method
- or adding a category for every test method  and removing the new methods (like `PipelineTestInvocationsNode8`) that invoke the tests

Reply:
That's true. I'll look either:

- removing theory / inlineData for every test method
- or adding a category for every test method  and removing the new methods (like `PipelineTestInvocationsNode8`) that invoke the tests

Reply:
Thanks Team, I've addressed with new changes.